smapply/tasks.py

```diff
@@ -21,3 +21,7 @@ def sync_all_users():
                 user.profile.save()
 
                 sync_hubspot_user(user.profile)
+        elif not profile.smapply_user_data:
```

Comment:
Should this just be `else:`?  Otherwise, if a profile exists and `profile.smapply_user_data` is set once, it won't be updated again.

Reply:
Changed to else.